### ISSUE TITLE
Latest AdMob causes java.lang.RuntimeException on startup

### ISSUE THREAD

**Issue by wombatwingdings** [FrogSquare/GodotFireBase#42]
Running latest clean build of Godot (3.1 branch) + GodotFireBase causes the following Java runtime exception on app startup in Android:
```
03-26 17:41:23.086  6605  6605 E AndroidRuntime: ******************************************************************************
03-26 17:41:23.086  6605  6605 E AndroidRuntime: * The Google Mobile Ads SDK was initialized incorrectly. AdMob publishers    *
03-26 17:41:23.086  6605  6605 E AndroidRuntime: * should follow the instructions here: https://goo.gl/fQ2neu to add a valid  *
03-26 17:41:23.086  6605  6605 E AndroidRuntime: * App ID inside the AndroidManifest. Google Ad Manager publishers should     *
03-26 17:41:23.086  6605  6605 E AndroidRuntime: * follow instructions here: https://goo.gl/h17b6x.                           *
03-26 17:41:23.086  6605  6605 E AndroidRuntime: ******************************************************************************
03-26 17:41:23.086  6605  6605 E AndroidRuntime:
03-26 17:41:23.086  6605  6605 E AndroidRuntime:
03-26 17:41:23.086  6605  6605 E AndroidRuntime:        at android.app.ActivityThread.installProvider(ActivityThread.java:6423)
03-26 17:41:23.086  6605  6605 E AndroidRuntime:        at android.app.ActivityThread.installContentProviders(ActivityThread.java:5972)
03-26 17:41:23.086  6605  6605 E AndroidRuntime:        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5893)
03-26 17:41:23.086  6605  6605 E AndroidRuntime:        at android.app.ActivityThread.-wrap1(Unknown Source:0)
03-26 17:41:23.086  6605  6605 E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1709)
03-26 17:41:23.086  6605  6605 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:105)
03-26 17:41:23.086  6605  6605 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:164)
03-26 17:41:23.086  6605  6605 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:6694)
03-26 17:41:23.086  6605  6605 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
03-26 17:41:23.086  6605  6605 E AndroidRuntime:        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
03-26 17:41:23.086  6605  6605 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:769)
03-26 17:41:23.086  6605  6605 E AndroidRuntime: Caused by: java.lang.IllegalStateException:
```
It seems that the latest Admob SDK requires the addition of APPLICATION_ID to the manifest:
```
<application>

      <meta-data 
           android:name="com.google.android.gms.ads.APPLICATION_ID" 
           android:value="ca-app-pub-################~##########"/>
</application>
```
I tried adding the following to the end of modules\firebase\android_src\AndroidManifestChunk.xml and it *fixed* the problem for me:
```
	<meta-data 
	android:name="com.google.android.gms.ads.APPLICATION_ID" 
	android:value="ca-app-pub-################~##########"/>
```
I'm not sure why no one else is getting this problem. Is this something that needs to be added to GodotFireBase?

Thanks.

**Comment by gravin333** [comment]
Thanks for you instruction. I got the same excaption and your solution works for me

**Comment by RameshRavone** [comment]
Fixed `78a70175db33b44600ffdd650a5597947cc6d6a3`

**Comment by wombatwingdings** [comment]
This is great. Very quick. Thank you very much!
